src/Components/ADT3DSceneBuilder/Internal/Behaviors/VisualRules/VisualRuleList.tsx

```diff
@@ -45,10 +45,36 @@ function getBadgesAndMeshesCount(item: IVisualRule, t: TFunction<string>) {
             meshColoringCount = meshColoringCount + 1;
         }
     });
-    const text = t('3dSceneBuilder.behaviorVisualRulesTab.meshAndBadgeCount', {
-        meshColoringCount: meshColoringCount,
-        badgeCount: badgeCount
-    });
+
+    return [badgeCount, meshColoringCount];
+}
+
+function getSecondaryText(item: IVisualRule, t: TFunction<string>) {
+    const [badgeCount, meshColoringCount] = getBadgesAndMeshesCount(item);
+    const badgeCondition =
+        badgeCount > 1
+            ? t('3dSceneBuilder.behaviorVisualRulesTab.multipleBadges', {
```

Comment:
we should really use multiple locale strings like I mentioned earlier. In languages other than english they might not concatenate these values together the same way. So we should have 3 locale strings, 1. Only badges, 2 only Color, 3. both

Reply:
i guess i'm not this comment, i have the locale string setup like 
![image](https://user-images.githubusercontent.com/80120000/196563304-5d91c4a5-5bae-48bf-b2a1-7acebd1c0c0c.png)

Reply:
yeah. but you concatenate them with a , which might not be valid in spanish or something.

Reply:
I know for sure in spanish it's valid but i get your point...so like should i include the comma in the locale strings??

Reply:
yeah. but thinking of the plural cases it gets kind of verbose. :( We should need 
1. single badge
2. multi badge
3. single color
4. multi color
5. single badge + multi color
6. multi badge + single color
7. multi badge + multi color

Reply:
So what you have is a good start, but then add the extra conditions for the combined strings